usermenu.php

```diff
@@ -169,6 +169,28 @@ function usermenu_civicrm_navigationMenu(&$menu) {
   _usermenu_civix_navigationMenu($menu);
 } // */
 
+/**
+ * Implements hook_civicrm_alterContent().
+ * Adds extra settings fields to the Civicase Admin Settings form.
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_alterContent/
+ */
+function usermenu_civicrm_alterContent (&$content, $context, $templateName, $form) {
```

Comment:
The last parameter is actually an object not necessarily a form object. Could be a page or form object. 
See: https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_alterContent/
We can change to `$object`